assets/pages/support-landing/components/patronsEvents.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+
+import PatronsEvents from 'components/patronsEvents/patronsEvents';
```

Comment:
Naming a non-shared component with the same name as a shared-component feels tangled. Is it possible to come up with a  better name? If not, Is it worth to consider to make the shared component a connected component?

Reply:
In that way, we will avoid going from file to file.

Reply:
Yeah I could call it `ConnectedPatronsEvents` instead, I wasn't sure which way would be clearer.

Reply:
Done